nvflare/fuel/utils/class_utils.py

```diff
@@ -80,24 +72,36 @@ def __init__(self, base_pkgs: List[str], module_names: List[str], exclude_libs=T
         self.base_pkgs = base_pkgs
         self.module_names = module_names
         self.exclude_libs = exclude_libs
+
+        self._logger = logging.getLogger(self.__class__.__name__)
         self._class_table = {}
         self._create_classes_table()
 
     def _create_classes_table(self):
         for base in self.base_pkgs:
             package = importlib.import_module(base)
 
-            for importer, modname, ispkg in pkgutil.walk_packages(path=package.__path__, prefix=package.__name__ + "."):
-
-                if modname.startswith(base):
-                    if not self.exclude_libs or (".libs" not in modname):
-                        if any(modname.startswith(base + "." + name + ".") for name in self.module_names):
+            for module_info in pkgutil.walk_packages(path=package.__path__, prefix=package.__name__ + "."):
+                module_name = module_info.name
+                if any(module_name.startswith(deprecated_package) for deprecated_package in DEPRECATED_PACKAGES):
+                    continue
+                if module_name.startswith(base):
+                    if not self.exclude_libs or (".libs" not in module_name):
+                        if any(module_name.startswith(base + "." + name + ".") for name in self.module_names):
                             try:
-                                module = importlib.import_module(modname)
+                                module = importlib.import_module(module_name)
                                 for name, obj in inspect.getmembers(module):
-                                    if inspect.isclass(obj) and obj.__module__ == modname:
-                                        self._class_table[name] = modname
-                            except (ModuleNotFoundError, RuntimeError):
+                                    if (
+                                        not name.startswith("_")
+                                        and inspect.isclass(obj)
+                                        and obj.__module__ == module_name
+                                    ):
+                                        self._class_table[name] = module_name
+                            except (ModuleNotFoundError, RuntimeError) as e:
+                                self._logger.warning(
```

Comment:
This will result to a lot warning messages when starting the server, client, and also the admin. This may not be the behaviors we want.

Reply:
It will only print warnings about the package that can't be imported, which should not be in app common in the first place. 

Like HE, PSI.

I will move them to app opt as well.